deploy/kubernetes/controller.yaml

```diff
@@ -23,6 +23,18 @@ rules:
   - apiGroups: [""]
     resources: ["events"]
     verbs: ["get", "list", "watch", "create", "update", "patch"]
+  - apiGroups: ["snapshot.storage.k8s.io"]
```

Comment:
We should create a separate clusterrole just for snapshotter, and tie it with the `csi-controller-sa` service account. Refer to https://github.com/kubernetes-csi/external-snapshotter/blob/01bd7f356e6718dee87914232d287631655bef1d/deploy/kubernetes/rbac.yaml